src/extension/inlineEdits/vscode-node/parts/vscodeWorkspace.ts

```diff
@@ -511,15 +501,32 @@ class VSCodeObservableTextDocument extends AbstractVSCodeObservableDocument impl
 	toRange(_textDocument: TextDocument, range: Range): Range | undefined {
 		return range;
 	}
+	async getCodeActions(offsetRange: OffsetRange, itemResolveCount: number, token: CancellationToken): Promise<CodeActionData[] | undefined> {
```

Comment:
Using OffSetRange as CodeActionData and existing DiagnosticData deals with Offset Ranges and not regular ranges.